src/lexer/tokenizer.rs

```diff
@@ -0,0 +1,415 @@
+use std::{process, rc::Rc, path::Path};
```

Comment:
`use super::*;`   😉

Reply:
wild imports seem really fishy, should we really not manually import that which we use each time?

Reply:
I mean ye can, just a thing I've used before

Reply:
I just let the lsp auto import for me and I clean up the unused ones later. If i use super::*, i would have to do logger::log_at type shit i dont like that personally.

Reply:
no no no ye dont have to do logger::whatevs
to me using super::*; more clearly shows what other imports are using than the rest of the project. still up to preference I guess